framework/doc/content/syntax/Mesh/splitting.md

```diff
@@ -51,9 +51,20 @@ It is noted that if there are mesh meta data generated by mesh generators, these
 will be written to a binary file under the generated directory that can be loaded when using
 split meshes.
 
+!alert note
+The mesh splitter commands do not work with DistributedMesh. You must only split with a ReplicatedMesh.
+
 ## Using Split Meshes
 
-To use a mesh split configuration use the `--use-split` flag (which takes no arguments):
+There are two ways of using referring to meshes in parallel in MOOSE, as split or as distributed. While there is no conceptual difference between the two there is a difference in implementation and features.
+
+### Loading a split mesh using [FileMeshGenerator.md]
+
+A split mesh, i.e. a `.cpr` file, may be loaded using a [FileMeshGenerator.md] if access to the mesh is needed before the full simulation case is set-up. In fact this approach is recommended; it leads to correct element ghosting whereas the approach we introduce in the following section may not.
```

Comment:
```suggestion
A split mesh, i.e. a `.cpr` file, may be loaded using a [FileMeshGenerator.md] if access to the mesh is needed before the full simulation case is set-up, notably for additional mesh generation with [mesh generators](syntax/Mesh/index.md#mesh-generators). In fact this approach is recommended; it leads to correct element ghosting whereas the approach we introduce in the following section may not.
```

Reply:
you ll have to add a `id=mesh-generators` to syntax/Mesh/index.md right near the beginning to make that work